site/content/blog/gsoc-2020-conclusion.md

```diff
@@ -0,0 +1,90 @@
++++
+date = "2020-09-09T15:00:00+00:00"
+title = "GSoC Projects Successfully Completed"
+author = "Sam Elliott and Pirmin Vogel"
++++
+
+Time is ticking and summer is almost over already. With that, also our this
+years’ [Google Summer of Code](https://summerofcode.withgoogle.com/) (GSoC)
+projects are coming to an end. A lot of open-source coding has been done, pull
+requests have been made, reviewed and merged. Experiments have been conducted,
+results were gathered, interpreted and presented. Bugs were found and fixed, and
+the resulting designs further improved. Both our students and mentors have been
+working hard and we are pleased to announce that both our two projects described
+below have been completed successfully.
+
+## Flavien Solt: Simulated Memory Controller
+
+It is a common pitfall to misinterpret or incorrectly scale performance numbers
+derived from benchmarks run on an FPGA-based SoC design. The problem is that the
+external memory interface is running at a very high speed compared to the core
+CPU (e.g. a 25MHz core clock speed but external memory running a several hundred
+MHz). This can be misleading when trying to consider what the performance would
+be on an ASIC, as the CPU clock speed could be many times higher but the memory
+frequency remains the same or increases by a much smaller amount. The solution
+is to implement a memory controller for the FPGA that can accurately “simulate”
+memory access delays in terms of the processor’s clock cycles.
+
+In this project, Flavien designed, implemented and verified such a Simulated
+Memory Controller IP core from scratch. The core features configurable AMBA AXI4
+host and device interfaces and is inserted between the processor and the real
+memory controller. Internally, it observes the memory requests sent by the
+processor, buffers the responses from the memory controller, and releases them
+to the processor after inserting an additional delay. The delays are determined
+by a separated Delay Calculator block that emulates a memory request scheduler.
+The first version designed by Flavien uses a First-Ready, First-Come First
+Served strategy but it can be replaced by more advanced schedulers if needed in
+the future. The design can be used both in RTL simulation and on FPGA.
+
+All of Flavien's code, including documentation, a Verilator testbench, and a
+working FPGA example design with a DDR memory can be found in [our
+`gsoc-sim-mem` GitHub repository](https://github.com/lowRISC/gsoc-sim-mem).
```

Comment:
@GregAC it would be great if you could have a quick look at this and make sure it accurately describes the current status of the codebase.